config/routes.rb

```diff
@@ -2,9 +2,9 @@
   root 'homepage#index'
   namespace :api do
     resources :orders, only: [:index, :create, :show]
-    resources :doughnuts
+    resources :doughnuts, only: [:index, :show, :create, :update, :destroy]
```

Comment:
Since we're no longer going to be doing anything related to having multiple sellers, we realistically only need the `index` action now. Delete every other method.

Reply:
Don't we still need the other actions so a user with the seller role can add, edit, and delete donuts?

Reply:
Oh I thought we decided we were not adding anymore features on the backend. But if not, I'm fine with keeping these actions as is except for `delete`, since that can cause some complications for how we should display `OrderItems` and `CartItems` if the associated `Doughnut` entity is ever deleted.